test/nirum_fixture/fixture/foo.nrm

```diff
@@ -100,3 +100,10 @@ record person (
 record people (
   {person} people
 );
+
+record product (
+    text name,
+    int64? price,
+    bool sale,
+    uri? url,
+);
```

Comment:
We should test union tags as well e.g.:

~~~~ nirum
union union-test = tag-a
                 | tag-b (text a, text? b, int64 c, int64 d?);
~~~~